### ISSUE TITLE
Unable to spawn isolate: Unhandled exception: Could not import "package:$pub/transformer_isolate.dart" 

### ISSUE THREAD

**Issue by zoechi** [dart-lang/pub#1409]
```
Dart VM version: 1.16.0-edge.e072ae33c6c58d925cd0ec48f0992d8c4d14affb (Tue Apr 19 15:07:53 2016) on "linux_x64"
```

With this project https://github.com/bwu-dart-playground/route-example-for-ie10

```
 (master) $ pub serve web --trace
Loading source assets... 
Loading angular2 and dart_to_js_script_rewriter transformers... 
Unable to spawn isolate: Unhandled exception:
Could not import "package:$pub/transformer_isolate.dart" from "file:///tmp/pub_73zar2/runInIsolate.dart": Failure getting http://localhost:34103/packages/$pub/transformer_isolate.dart:
  500 Internal Server Error
#0      _asyncLoadErrorCallback (dart:_builtin:162)
#1      _asyncLoadError (dart:_builtin:584)
#2      _handleLoaderReply (dart:_builtin:401)
#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:148)
dart:isolate-patch/isolate_patch.dart      Isolate.spawnUri.<async>
dart:async-patch/async_patch.dart 34       _asyncErrorWrapperHelper.<fn>
dart:async/zone.dart 1142                  _RootZone.runBinary
dart:async/future_impl.dart 581            _Future._propagateToListeners.handleError
dart:async/future_impl.dart 643            _Future._propagateToListeners
dart:async/future_impl.dart 434            _Future._completeError
dart:async/future_impl.dart 56             _SyncCompleter._completeError
dart:async/future_impl.dart 27             _Completer.completeError
dart:isolate-patch/isolate_patch.dart 439  Isolate._spawnCommon.<fn>
dart:isolate-patch/isolate_patch.dart 148  _RawReceivePortImpl._handleMessage
---- Log transcript ----
FINE: Pub 1.16.0-edge.e905d64058cfc07c8f5f8e2c048f8e040774d41e
FINE: Loading asset environment...
FINE: Initializing barback...
FINE: Serving packages on localhost:0.
FINE: Providing sources for router_ie_10|lib.
FINE: Loading source assets finished (0.0s).
FINE: Provided sources.
FINE: Loading transformers...
FINE: Transformer dependencies:
    | angular2: -
    | dart_to_js_script_rewriter: -
FINE: Cached transformer snapshot is out-of-date, deleting.
FINE: Loading transformers from [angular2, dart_to_js_script_rewriter]
IO  : Created temp directory /tmp/pub_73zar2
IO  : Writing 254 characters to text file /tmp/pub_73zar2/runInIsolate.dart.
FINE: Instance of 'BarbackServer' GET packages/$pub/transformer_isolate.dart
    | 'package:pub/src/barback/barback_server.dart': malformed type: line 166: type 'SHA1' is not loaded
    |       var sha = new SHA1();
    |                     ^
    | 
    | 
    | dart:core-patch/errors_patch.dart 56                       _TypeError._throwNew
    | package:pub/src/barback/barback_server.dart 166            BarbackServer._serveAsset.<async>
    | dart:async-patch/async_patch.dart 27                       _asyncThenWrapperHelper.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 134  StackZoneSpecification.registerUnaryCallback.<fn>.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 205  StackZoneSpecification._run
    | package:stack_trace/src/stack_zone_specification.dart 134  StackZoneSpecification.registerUnaryCallback.<fn>
    | dart:async/zone.dart 902                                   _rootRunUnary
    | dart:async/zone.dart 798                                   _CustomZone.runUnary
    | dart:async/future_impl.dart 553                            _Future._propagateToListeners.handleValueCallback
    | dart:async/future_impl.dart 639                            _Future._propagateToListeners
    | dart:async/future_impl.dart 426                            _Future._completeWithValue
    | dart:async/future_impl.dart 481                            _Future._asyncComplete.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 205  StackZoneSpecification._run
    | package:stack_trace/src/stack_zone_specification.dart 124  StackZoneSpecification.registerCallback.<fn>
    | dart:async/zone.dart 895                                   _rootRun
    | dart:async/zone.dart 790                                   _CustomZone.run
    | dart:async/zone.dart 696                                   _CustomZone.runGuarded
    | dart:async/zone.dart 721                                   _CustomZone.bindCallback.<fn>
    | dart:async/schedule_microtask.dart 41                      _microtaskLoop
    | dart:async/schedule_microtask.dart 50                      _startMicrotaskLoop
    | dart:isolate-patch/isolate_patch.dart 96                   _runPendingImmediateCallback
    | dart:isolate-patch/isolate_patch.dart 149                  _RawReceivePortImpl._handleMessage
FINE: Instance of 'BarbackServer' GET packages/dart_to_js_script_rewriter/dart_to_js_script_rewriter.dart
    | 'package:pub/src/barback/barback_server.dart': malformed type: line 166: type 'SHA1' is not loaded
    |       var sha = new SHA1();
    |                     ^
    | 
    | 
    | dart:core-patch/errors_patch.dart 56                       _TypeError._throwNew
    | package:pub/src/barback/barback_server.dart 166            BarbackServer._serveAsset.<async>
    | dart:async-patch/async_patch.dart 27                       _asyncThenWrapperHelper.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 134  StackZoneSpecification.registerUnaryCallback.<fn>.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 205  StackZoneSpecification._run
    | package:stack_trace/src/stack_zone_specification.dart 134  StackZoneSpecification.registerUnaryCallback.<fn>
    | dart:async/zone.dart 902                                   _rootRunUnary
    | dart:async/zone.dart 798                                   _CustomZone.runUnary
    | dart:async/future_impl.dart 553                            _Future._propagateToListeners.handleValueCallback
    | dart:async/future_impl.dart 639                            _Future._propagateToListeners
    | dart:async/future_impl.dart 426                            _Future._completeWithValue
    | dart:async/future_impl.dart 481                            _Future._asyncComplete.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 205  StackZoneSpecification._run
    | package:stack_trace/src/stack_zone_specification.dart 124  StackZoneSpecification.registerCallback.<fn>
    | dart:async/zone.dart 895                                   _rootRun
    | dart:async/zone.dart 790                                   _CustomZone.run
    | dart:async/zone.dart 696                                   _CustomZone.runGuarded
    | dart:async/zone.dart 721                                   _CustomZone.bindCallback.<fn>
    | dart:async/schedule_microtask.dart 41                      _microtaskLoop
    | dart:async/schedule_microtask.dart 50                      _startMicrotaskLoop
    | dart:isolate-patch/isolate_patch.dart 96                   _runPendingImmediateCallback
    | dart:isolate-patch/isolate_patch.dart 149                  _RawReceivePortImpl._handleMessage
FINE: Instance of 'BarbackServer' GET packages/angular2/transformer.dart
    | 'package:pub/src/barback/barback_server.dart': malformed type: line 166: type 'SHA1' is not loaded
    |       var sha = new SHA1();
    |                     ^
    | 
    | 
    | dart:core-patch/errors_patch.dart 56                       _TypeError._throwNew
    | package:pub/src/barback/barback_server.dart 166            BarbackServer._serveAsset.<async>
    | dart:async-patch/async_patch.dart 27                       _asyncThenWrapperHelper.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 134  StackZoneSpecification.registerUnaryCallback.<fn>.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 205  StackZoneSpecification._run
    | package:stack_trace/src/stack_zone_specification.dart 134  StackZoneSpecification.registerUnaryCallback.<fn>
    | dart:async/zone.dart 902                                   _rootRunUnary
    | dart:async/zone.dart 798                                   _CustomZone.runUnary
    | dart:async/future_impl.dart 553                            _Future._propagateToListeners.handleValueCallback
    | dart:async/future_impl.dart 639                            _Future._propagateToListeners
    | dart:async/future_impl.dart 426                            _Future._completeWithValue
    | dart:async/future_impl.dart 481                            _Future._asyncComplete.<fn>
    | package:stack_trace/src/stack_zone_specification.dart 205  StackZoneSpecification._run
    | package:stack_trace/src/stack_zone_specification.dart 124  StackZoneSpecification.registerCallback.<fn>
    | dart:async/zone.dart 895                                   _rootRun
    | dart:async/zone.dart 790                                   _CustomZone.run
    | dart:async/zone.dart 696                                   _CustomZone.runGuarded
    | dart:async/zone.dart 721                                   _CustomZone.bindCallback.<fn>
    | dart:async/schedule_microtask.dart 41                      _microtaskLoop
    | dart:async/schedule_microtask.dart 50                      _startMicrotaskLoop
    | dart:isolate-patch/isolate_patch.dart 96                   _runPendingImmediateCallback
    | dart:isolate-patch/isolate_patch.dart 149                  _RawReceivePortImpl._handleMessage
IO  : Deleting directory /tmp/pub_73zar2.
FINE: Loading angular2 and dart_to_js_script_rewriter transformers finished (0.2s).
FINE: Loading transformers finished (0.4s).
FINE: Initializing barback finished (0.5s).
FINE: Loading asset environment finished (0.6s).
ERR : Unable to spawn isolate: Unhandled exception:
    | Could not import "package:$pub/transformer_isolate.dart" from "file:///tmp/pub_73zar2/runInIsolate.dart": Failure getting http://localhost:34103/packages/$pub/transformer_isolate.dart:
    |   500 Internal Server Error
    | #0      _asyncLoadErrorCallback (dart:_builtin:162)
    | #1      _asyncLoadError (dart:_builtin:584)
    | #2      _handleLoaderReply (dart:_builtin:401)
    | #3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:148)
    | dart:isolate-patch/isolate_patch.dart      Isolate.spawnUri.<async>
    | dart:async-patch/async_patch.dart 34       _asyncErrorWrapperHelper.<fn>
    | dart:async/zone.dart 1142                  _RootZone.runBinary
    | dart:async/future_impl.dart 581            _Future._propagateToListeners.handleError
    | dart:async/future_impl.dart 643            _Future._propagateToListeners
    | dart:async/future_impl.dart 434            _Future._completeError
    | dart:async/future_impl.dart 56             _SyncCompleter._completeError
    | dart:async/future_impl.dart 27             _Completer.completeError
    | dart:isolate-patch/isolate_patch.dart 439  Isolate._spawnCommon.<fn>
    | dart:isolate-patch/isolate_patch.dart 148  _RawReceivePortImpl._handleMessage
FINE: Exception type: CrossIsolateException
FINE: package:pub/src/dart.dart 189                          runInIsolate.<async>.<fn>.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             _Completer.completeError
    | package:pub/src/dart.dart 206                          runInIsolate.<async>.<fn>.<async>
    | ===== asynchronous gap ===========================
    | package:pub                                            runInIsolate.<async>.<fn>
    | package:pub/src/io.dart 854                            withTempDir.<fn>.<fn>
    | dart:async                                             Future.Future.sync
    | package:pub/src/io.dart 854                            withTempDir.<fn>
    | dart:async                                             Future.Future.sync
    | package:pub/src/io.dart 852                            withTempDir
    | package:pub/src/dart.dart 175                          runInIsolate.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             Future.Future.microtask
    | package:pub/src/dart.dart                              runInIsolate
    | package:pub/src/barback/transformer_isolate.dart 76    TransformerIsolate.spawn.<fn>
    | dart:async                                             _Completer.completeError
    | package:pub/src/barback/pub_package_provider.dart 109  PubPackageProvider.getAsset.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             _Future.then
    | package:pub/src/barback/transformer_isolate.dart 55    TransformerIsolate.spawn
    | package:pub/src/barback/transformer_loader.dart 55     TransformerLoader.load.<async>.<fn>
    | package:pub/src/log.dart 407                           progress
    | package:pub/src/barback/transformer_loader.dart 54     TransformerLoader.load.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             Future.Future.microtask
    | package:pub/src/barback/transformer_loader.dart        TransformerLoader.load
    | package:pub/src/barback/load_all_transformers.dart 85  loadAllTransformers.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             Future.Future.microtask
    | package:pub/src/barback/load_all_transformers.dart     loadAllTransformers
    | package:pub/src/barback/asset_environment.dart 511     AssetEnvironment._load.<fn>.<async>.<fn>.<async>.<fn>.<async>
    | ===== asynchronous gap ===========================
    | package:pub                                            AssetEnvironment._load.<fn>.<async>.<fn>.<async>.<fn>
    | package:pub/src/log.dart 407                           progress
    | package:pub/src/barback/asset_environment.dart 510     AssetEnvironment._load.<fn>.<async>.<fn>.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             Future.Future.sync
    | package:pub/src/barback/asset_environment.dart 670     AssetEnvironment._withStreamErrors
    | package:pub/src/barback/asset_environment.dart 509     AssetEnvironment._load.<fn>.<async>
    | dart:async                                             _SyncCompleter.complete
    | package:pub/src/barback/asset_environment.dart 671     AssetEnvironment._withStreamErrors.<fn>
    | dart:async                                             _SyncCompleter.complete
    | package:pub/src/barback/asset_environment.dart 530     AssetEnvironment._provideSources.<async>
    | dart:async                                             _SyncCompleter.complete
    | package:pub/src/barback/asset_environment.dart 529     AssetEnvironment._provideSources.<async>.<fn>.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             _asyncThenWrapperHelper
    | package:pub/src/barback/asset_environment.dart         AssetEnvironment._load.<fn>
    | package:pub/src/log.dart 407                           progress
    | package:pub/src/barback/asset_environment.dart 458     AssetEnvironment._load
    | package:pub/src/barback/asset_environment.dart 82      AssetEnvironment.create.<fn>.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             Future.Future.microtask
    | package:pub/src/barback/asset_environment.dart         AssetEnvironment.create.<fn>
    | package:pub/src/log.dart 407                           progress
    | package:pub/src/barback/asset_environment.dart 74      AssetEnvironment.create
    | package:pub/src/command/serve.dart 93                  ServeCommand.onRunTransformerCommand.<async>
    | ===== asynchronous gap ===========================
    | dart:async                                             Future.Future.microtask
    | package:pub/src/command/serve.dart                     ServeCommand.onRunTransformerCommand
    | package:pub/src/command/barback.dart 56                BarbackCommand.run
    | package:args/command_runner.dart 176                   CommandRunner.runCommand.<fn>
    | dart:async                                             Future.Future.sync
    | package:args/command_runner.dart 129                   CommandRunner.runCommand
    | package:pub/src/command_runner.dart 131                PubCommandRunner.runCommand.<async>.<fn>
    | dart:async                                             Future.Future.sync
    | package:pub/src/utils.dart 113                         captureErrors.<fn>
    | package:stack_trace                                    Chain.capture
    | package:pub/src/utils.dart 127                         captureErrors
    | package:pub/src/command_runner.dart 131                PubCommandRunner.runCommand.<async>
    | dart:async                                             _SyncCompleter.complete
    | package:pub/src/command_runner.dart 217                PubCommandRunner._validatePlatform.<async>
---- End log transcript ----
```

**Comment by alexander-doroshko** [comment]
I'm getting the same with SDK 1.16.0-dev.5.3 for Windows x64 that I downloaded from [here](http://gsdview.appspot.com/dart-archive/channels/dev/raw/1.16.0-dev.5.3/sdk/):
```
>pub get
Resolving dependencies...
Got dependencies!
Precompiling dependencies...
Loading source assets...
Loading angular2/transform/codegen, dart_to_js_script_rewriter, observe and smoke/src/default_transformer transformers...
Unable to spawn isolate: Unhandled exception:
Could not import "package:$pub/transformer_isolate.dart" from "file:///C:/Users/alexander.doroshko/AppData/Local/Temp/pub_dbc9b2b9-0710-11e6-8283-e82aea7e7d3a/runInIsolate.dart": Failure getting http://localhost:60457/packages/$pub/transformer_isolate.dart:
  500 Internal Server Error
#0      _asyncLoadErrorCallback (dart:_builtin:162)
#1      _asyncLoadError (dart:_builtin:584)
#2      _handleLoaderReply (dart:_builtin:401)
#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:148)
dart:isolate-patch/isolate_patch.dart      Isolate.spawnUri.<async>
dart:async-patch/async_patch.dart 34       _asyncErrorWrapperHelper.<fn>
dart:async/zone.dart 1142                  _RootZone.runBinary
dart:async/future_impl.dart 581            _Future._propagateToListeners.handleError
dart:async/future_impl.dart 643            _Future._propagateToListeners
dart:async/future_impl.dart 434            _Future._completeError
dart:async/future_impl.dart 56             _SyncCompleter._completeError
dart:async/future_impl.dart 27             _Completer.completeError
dart:isolate-patch/isolate_patch.dart 439  Isolate._spawnCommon.<fn>
dart:isolate-patch/isolate_patch.dart 148  _RawReceivePortImpl._handleMessage
```

**Comment by munificent** [comment]
I believe this is fixed by: 04ff0cc2cb6a3b698159b1aeef14cd3d7c90e287

If not, let me know and I'll re-open.
